.github/workflows/deploy-gcp-tests.yml

```diff
@@ -234,7 +239,38 @@ jobs:
           --zone ${{ env.ZONE }}
           sleep 60
 
+      # SSH into the just created VM, and create a Docker container to run the incoming test 
+      # from ${{ inputs.test_id }}, then create a docker volume with the recently attached disk.
+      # The disk will be mounted in ${{ inputs.root_state_path }}/${{ inputs.zebra_state_dir }}.
+      #
+      # This step mounts the volume only when a single cached state is needed, in this case
+      # the cached state from Zebra.
+      - name: Run ${{ inputs.test_id }} test
```

Comment:
This step has the same name as the next step

Reply:
Yes, but as the name is dynamic, that should not be a problem. It's also easier later for branch protection.